tests/test_sims.py

```diff
@@ -575,6 +575,37 @@ def test_sbm_multiwt_undirected_loopy(self):
         self.assertTrue(A.shape == (np.sum(self.n), np.sum(self.n)))
         pass
 
+    def test_sbm_dc_dcargs_directed_loopy(self):
```

Comment:
I don't understand the point of this test, looks like you're generating the dc weights with a function and then passing the scalar values into the sbm code. I thought sbm code could take the function itself as an argument? Also, why remove the test that you already had?

As written, can you only pass in one function and one set of kwargs if using the function option for the dc weights?

Reply:
Yes, you can only pass in one function/set of kwargs. I was debating modifying it so that you can also pass a function for the vertices in each block?

I generate the dc weights outside of the function because if generated internally, I don't know a priori know the theoretical degrees that I should see for each vertex. But I can formulate an additional test that passes the function.